### DISCUSSION TITLE
How to best activate Quick Switcher++ on mobile?

### DISCUSSION THREAD

**Discussion by pmeyers** [darlal/obsidian-switcher-plus#117] [Q&A] [answered]
I've enabled Quick Switcher++ and am very happily using it on desktop. A lot of super useful functionality that is very handy.

But I'm not sure how to quickly activate it on mobile? If I press the middle (+) button at the bottom of the screen on mobile the default core Quick Switcher opens. I thought by enabling "Override Standard mode behavior" it would inject the Quick Switcher++ functionality/styling into the default Quick Switcher?

> Override Standard mode behavior: Switcher++ will change the default Obsidian builtin Switcher functionality (Standard mode) to inject custom behavior. Custom behavior includes features like enhance rendering of suggestion items to display additional information, special path display handling, and tab navigation features.

If I open the command palette I can search for Quick Switcher++ and open it that way, but I was hoping to have quicker access.

I'm relatively new to Obsidian, so I could be overlooking something super obvious.

**Comment by darlal** [comment, marked as answer]
Hey @pmeyers glad you're finding the plugin useful. Overriding the (+) button on mobile is on the list of things to look into, but is still on the todo list unfortunately. Even without that, there are several good options to consider:

- **Set your preferred Switcher++ mode as the "Mobile Quick Action"** 
This makes it so that when you pull down from the top of the screen, it will trigger Switcher++. This can be configured from the "Mobile" tab in app settings. For example, you could make Switcher++ command mode the default, and replace the builtin command palette - now you can easily switch to any of the Switcher++ modes easily.
- **Add Switcher++ modes to the ribbon menu**
This makes it so that Switcher++ commands appear in the ribbon menu (the right most button at the bottom of the screen). You can add as many of the Switcher++ modes there as you want. This can be configured in the Switcher++ "General Settings" section, "Show ribbon icons".
- **Add Switcher++ modes to the keyboard toolbar**
This makes it so that when the mobile keyboard is displayed, icons for Switcher++ commands appear in the mobile keyboard toolbar. You can add as many of the Switcher++ modes there as you want. This can also be configured from the "Mobile" tab in app settings. Scroll all the way down to the "Add global command" section.

**Reply by pmeyers** [reply]
This is exactly what I was looking for, thanks so much for the in-depth reply.

Setting Switcher++ as the Mobile Quick Action actually feels like a quicker and more useful solution than using the bottom button so this is great. I'm loving that it's automatically opening my results in new tabs, this is going to make my mobile Obsidian usage much better.

I forgot that the full ribbon is displayed when long pressing the right most button, so thanks for the reminder on that. I've added all of the modes I've been using on desktop there.

This is really great and I love all of the functionality Switcher++ adds. Makes Obsidian so much more pleasant to use. Top tier plugin. Thanks for taking the time to explain everything, I should have explored more and I would have eventually realized all of my answers were already in the settings. :)
